fiftyone/utils/ptv3.py

```
@@ -0,0 +1,309 @@
+        grid_size (0.025): the voxel grid size, in meters, used to serialize the
+            point cloud before inference. Must match the value the checkpoint
+            was trained with
+        feature_keys (("coord", "strength")): the per-point input features the
+            backbone consumes, in order. ``"coord"`` expands to the three xyz
+            columns; any other key consumes one column of the input cloud. The
+            total width must equal the backbone's ``in_channels``
+        device (None): the device to use, e.g. ``"cuda"`` or ``"cpu"``. If not
+            provided, GPU is used when available
+    """
+
+    def __init__(self, d):
```

Ruff (0.15.17)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)